source/transactions/tests/pin-mongos.yml

```diff
@@ -443,7 +443,7 @@ tests:
     clientOptions:
       # Increase heartbeatFrequencyMS to avoid the race condition where an in
       # flight heartbeat refreshes the first mongoes' SDAM state in between
-      # the initial commitTransaction and the retry attempt.
+      # the insert connection error and the single commit attempt.
       heartbeatFrequencyMS: 5000
```

Comment:
Isn't the default heartbeat frequency 10,000 ms, in which case this is actually decreasing it?  Maybe we should bump it up to 30,000 ms?

Reply:
Yeah that's a very good point.